packages/frontend/src/shared/ui/preloader/Preloader.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from './Preloader.module.scss'
+export function Preloader (): JSX.Element {
+  return (
+    <div className={ styles['wrapper'] }>
+      <div className={ styles['preloader'] } />
+      <div className={ styles['preloader-r'] } />
+    </div>
```

Comment:
Для стилей используй запись через точку, где это возможно (редактор поидее сам подскажет где нужны скобки)
```suggestion
    <div className={ styles.wrapper }>
      <div className={ styles.preloader } />
      <div className={ styles.preloader-r } />
    </div>
```